unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/Interfaces/EntityIdHelper.cs

```diff
@@ -14,6 +14,11 @@ public class EntityIdHelper
         internal Dictionary<string, long> negativeValues = new Dictionary<string, long>();
 
         private int negativeCounter = -1;
+        private Regex regex;
+        public EntityIdHelper()
+        {
+            regex = new Regex("^*[A-Z]",RegexOptions.Compiled);
```

Comment:
base 36 is not `[A-Z0-9]` instead of `[A-Z]`?

Reply:
Base 36 is lowercases and numbers so we are finding if the id has a uppercase to take into account that this is not a base 36